wxt_cidrbot/git_webhook_handler.py

```diff
@@ -100,6 +99,112 @@ def webhook_request(self, event):
 
                 self.Api.messages.create(self.room_id, markdown=message_uninstall)
 
+        elif x_event_type in ('issues', 'pull_request'):
+            if event_action == 'opened':
+                self.triage_issue(installation_id, json_string, x_event_type)
+
+    # Assign a new github issue/pr on webhook
+    def triage_issue(self, installation_id, json_string, x_event_type):
+        event_info = self.check_installation(installation_id)
+        issue_type = "Pull request"
+        query_key = "pull_request"
+
+        # Issues and prs have different dict structures
+        if x_event_type == 'issues':
+            issue_type = "Issue"
+            query_key = "issue"
+            issue_num = json_string['issue']['number']
+        else:
+            issue_num = json_string['number']
+
+        room_id = event_info[0]['room_id']
+        issue_title = json_string[query_key]['title']
+        issue_url = json_string[query_key]['url']
+        issue_user = json_string[query_key]['user']['login']
+        repo_name = json_string['repository']['full_name']
+        repo_url = json_string['repository']['html_url']
+
+        hyperlink_format = f'<a href="{issue_url}">{issue_title}</a>'
+        hyperlink_format_repo = f'<a href="{repo_url}">{repo_name}</a>'
+        message = f"{issue_type} {hyperlink_format} created in {hyperlink_format_repo}. Performing automated triage:"
+
+        try:
+            triage = self.dynamo.get_triage(room_id)
+            repos = self.dynamo.get_repositories(room_id)
+        except Exception:
+            self.logging.debug("Error retrieving triage users and/or repos")
+            sys.exit(1)
+
+        if len(triage) < 1:
+            self.logging.debug("No triage users, quitting triage")
+            sys.exit(1)
+
+        URL = f'https://webexapis.com/v1/messages'
+        headers = {'Authorization': 'Bearer ' + self.wxt_access_token, 'Content-type': 'application/json;charset=utf-8'}
+        post_message = {'roomId': room_id, 'markdown': message}
+        response = requests.post(URL, json=post_message, headers=headers)
+        if response.status_code == 200:
+            self.logging.debug("Message created successfully")
+            msg_edit_id = json.loads(str(response.text))["id"]
+        else:
+            self.logging.debug("Status code %s | text %s", str(response.status_code), str(response.text))
+
+        session = requests.Session()
+        self.logging.debug("Starting triage")
+        author_list = []
+        user_issue_count = []
+
+        for triage_list_user in triage:
```

Comment:
likewise, if you rename triage to triage_list, then it would be more concise to make your loop variable `triage_user` or even just `user`

Reply:
Renamed to user